src/client/index.js

```diff
@@ -11,7 +11,7 @@
 // We use HTTP POST requests with CSRF Tokens to protect against CSRF attacks.
 
 /* global fetch:false */
-import { useState, useEffect, useContext, createContext, createElement } from 'react'
```

Comment:
With this, we can prevent future line changes in commits. (Idea: https://epicreact.dev/importing-react-through-the-ages/)